From 21840568911ded5fd7eff52d8e9b1f7bc4828528 Mon Sep 17 00:00:00 2001
From: Bai Ping <b51503@freescale.com>
Date: Wed, 9 Sep 2015 02:33:53 +0800
Subject: [PATCH 0430/5242] MLK-11521 ARM: dts: fix a typo for clock name
 'pll1_bypass_src'

commit  9cff4e961f832aaad5660ff9e53e6d139e2467e8 from
https://source.codeaurora.org/external/imx/linux-imx.git

Fix a typo for clock name 'pll1_bypass_src'.

Signed-off-by: Bai Ping <b51503@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6dl.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6dl.dtsi b/arch/arm/boot/dts/imx6dl.dtsi
index 2bee13e..f41eb94 100644
--- a/arch/arm/boot/dts/imx6dl.dtsi
+++ b/arch/arm/boot/dts/imx6dl.dtsi
@@ -43,7 +43,7 @@
 				 <&clks IMX6QDL_PLL1_BYPASS_SRC>;
 			clock-names = "arm", "pll2_pfd2_396m", "step",
 				      "pll1_sw", "pll1_sys", "pll1",
-				      "pll1_bypass", "Pll1_bypass_src";
+				      "pll1_bypass", "pll1_bypass_src";
 			arm-supply = <&reg_arm>;
 			pu-supply = <&reg_pu>;
 			soc-supply = <&reg_soc>;
-- 
1.7.9.5

